library/libobject.mli

```diff
@@ -72,7 +72,7 @@ type 'a substitutivity =
 
 type object_name = full_path * Names.KerName.t
 
-type open_filter = Unfiltered | Names of Globnames.ExtRefSet.t
```

Comment:
Not sure it's very useful too keep this around. You should purge all functions taking this as a dummy argument maybe?